src/main/resources/config.yml

```diff
@@ -108,240 +110,263 @@ whole-number-prices-only: false
+      example: 20 #Players with the quickshop.example permission can create 20 shops.
 
-#A list of block (materials) which can be used to create shops.
+#The list of blocks that can be used to create shops.
 #By default, chests are added to this list.
-#This will only work for blocks which implement InventoryHolder
-#in other words, no enderchest shops, no shops on dirt blocks, etc.
-#May cause unexpected behavior with some blocks... Eg don't make a
-#shop on a hopper (It sucks the display item in) and furnace shops
-#allow players to put whatever item they want in all 3 slots and
-#dispenser shops aren't protected from redstone... Etc.
+#This will only work for blocks that implement an InventoryHolder!
+#In other words, You cannot create shops with normal blocks like dirt or stone.
+#Enderchests can only be used as shop blocks when the "openinv" plugin is installed!
+#May cause unexpected behavior with some blocks...eg:
+#-Hopper shops suck the display item in, 
+#-Furnace shops allow players whatever item they want in all 3 slots,
+#-Dispenser shops aren't protected from redstone,
+#-Etc...
 shop-blocks:
   - CHEST
   - TRAPPED_CHEST
   - ENDER_CHEST
+
 shop:
-  #Cost to make a shop
+  #The cost to make a shop.
   cost: 10
-  #Should we refund the player when their shops are deleted/removed/broken?
+  #Should QS refund the player when their shops are deleted/removed/broken?
   refund: false
-  #Should we send stock message to staffs?
+  #Should QS send stock messages to staffs?
   sending-stock-message-to-staffs: false
-  #Should we disable trading for creative mode?
+  #Should QS disable trading in creative mode?
   disable-creative-mode-trading: false
-  #Should we disable super tool?(It's a tool which allow you break shop in creative mode by gold axe)
+  #Should QS disable the super tool?
+  #The Super Tool is a Golden Axe that allows creative mode players to break shops.
   disable-super-tool: false
-  #Should we allow the owner of shop break the sign (for sign replacement, but may cause sign dupe when auto-sign is true)?
+  #Should QS allow the owner of a shop to break the sign to let players replace the sign?
+  #This may cause a sign dupe when auto-sign is true!
   allow-owner-break-shop-sign: false
 
-  #Is there a fee for changing prices on a shop (Help deter endless undercutting)
+  #Should there be a fee for changing shop prices?
+  #This can help with endless price undercutting.
   price-change-requires-fee: true
-  #If price changes require a fee, how much is the fee
+  #The amount of this fee.
   fee-for-price-change: 50
 
-  #Should we try and lock their shops from other players, so people can't steal from them?
+  #Should QS try to lock shops for other players?
+  #This prevents players from stealing from them.
   lock: true
 
-  #Should we disable the quick way of creating shop? (click a chest with item)
+  #Should QS disable the quick creation of shop?
+  #To quickly create a shop click a chest with an Item.
```

Comment:
To quickly create a shop by clicking a chest with an Item?

Reply:
"(click a chest with item)" -> "To quickly create a shop click a chest with an Item."

Improved explanation of what "the quick way of creating shop" is...